tmva/sofie/inc/TMVA/ROperator_Comparision.hxx

```diff
@@ -77,7 +77,7 @@ public:
    ROperator_Comparision(const std::string & nameX1, const std::string & nameX2, const std::string & nameY):
```

Comment:
@lmoneta do we need the `std::vector<Dim> fDimShapeX1;
   std::vector<Dim> fDimShapeX2;` here? They seem to be only used in two `if` cases